### DISCUSSION TITLE
Historical places and gazetteer links: finding a coherent strategy

### DISCUSSION THREAD

**Discussion by rybesh** [periodo/periodo-data#96] [Spatial coverage]
Working on the [Getty/DAI periods](https://github.com/periodo/periodo-data/issues/78), it is becoming clear to me that we need to decide on a consistent way to link to gazetteers when the textual spatial coverage description of a period is given as a historical region/period.

At first I had thought that we would just [use the existing DAI gazetteer links](https://github.com/periodo/periodo-places/issues/17) for the Getty/DAI periods, but:
1. a lot of the linked DAI gazetteer records duplicate gazetteer entries we already have, and
2. in many cases—especially for historical places—the DAI gazetteer entry is not so useful.

So, I'm finding myself having to make a lot of difficult decisions about how to import these periods in a way that is consistent with what we already have—and that's making me realize that we haven't been all that consistent so far.

To make this clearer, let's consider a specific example:

Getty defines the [Lan Xang](http://vocab.getty.edu/page/aat/300019196) period, which “refers to the style and period surrounding the culture that flourished in the Cambodian kingdom of Lan Xang (1353-1707 CE).” DAI has [enriched the Getty data](https://chronontology.dainst.org/period/nFpEeMIwIg3W) by extracting the years `1352` (why `1352` and not `1353`? 🤷‍♂️) and `1707` and linking to [the DAI gazetteer record for Lan Xang Kingdom](http://gazetteer.dainst.org/place/2379466). That gazetteer record provides some alternative names for Lan Xang Kingdom and tells us it is in Asia, but nothing more: no spatial location information. The corresponding [Wikidata item for Lan Xang](http://www.wikidata.org/entity/Q853477) has a lot more useful information and links elsewhere.

This is typical of many of the Getty/DAI records: the period term refers to a historical culture/kingdom/empire, and the “spatial coverage” is given as a term nearly identical to the period term. “Lan Xang was in Lan Xang Kingdom which was in Asia.” “Seleucid was in Seleucid Empire which was in Asia.” Etc.

So, how to bring a period like this into PeriodO? I see a few options:

1. Link the period to `wikidata:Lan Xang` in our [historical places gazetteer](https://github.com/periodo/periodo-places#historical-places-gazetteer) (which would first require adding [Lan Xang](http://www.wikidata.org/entity/Q853477) to that gazetteer).
    👍 The Lan Xang Wikidata item has useful info and links.
    👍 This more closely represents the way Getty/DAI represents the spatial coverage.
    👎 We have no spatial footprint we can show on a map for Lan Xang. 
    👎 People using the client to search for periods in Southeast Asia, Indochina, or Laos won’t find it, unless we add hierarchical relations to our gazetteers, and rewrite the spatial coverage filtering in the client to take these relations into account.
    👎 Arguably, the “Lan Xang” period and the “Lan Xang Kingdom” are the same exact thing. We’re not indicating spatial coverage in a useful way, we’re just linking to a close match for the concept in Wikidata.
    👎 We have to do some extra work to add a record to our historical places gazetteer. If we do this enough, eventually the historical places gazetteer could become quite large.

2. Link the period to `wikidata:Laos` in our [countries gazetteer](https://github.com/periodo/periodo-places#countries-gazetteer).
    👍 We already have Laos in our existing gazetteer, and we can show its footprint on a map.
    👍 People using the client to search for periods in Laos will find it.
    👎 Less fidelity to the source (though we can still have “Lan Xang Kingdom” as the textual spatial coverage description).
    👎 We have to decide what present-day administrative locations to link to. In this case, it seems pretty straightforward to use Laos, but the Getty description calls it a “Cambodian” kingdom, so maybe it should link to Cambodia too? Unfortunately it is rarely straightforward to determine the present-day equivalents of historical places in an automated way.

3. Link the period to `wikidata:Indochina` in our [geographic regions gazetteer](https://github.com/periodo/periodo-places#geographic-regions-gazetteer). While Laos is the “closest match” spatial-thing-we-have-boundaries-for,  Indochina is the “smallest containing” spatial-thing-we-have-boundaries-for. This basically has the same pros and cons as option 2, but avoids the Laos vs. Cambodia issue.

4. Do some combination of the above, i.e. link to all three things, or some two out of the three.

Given the number of cons for option 1, I’m inclined to go with 2 or 3, or a combination of 2 and 3. This creates some work to map historical places to modern equivalents, but that’s less work than having to constantly update the historical places gazetteer. I’m not sure whether a “closest match” or a “smallest containing” strategy is better. Should “Seleucid Empire” be `wikidata:Syria, wikidata:Iran, wikidata:Iraq, wikidata:Lebanon, wikidata:Israel, wikidata:State of Palestine, wikidata:Turkey` or `wikidata:Middle East`, or both?

If we do decide to go with something exclusive of option 1, then I would like to phase out our historical gazetteer. I only created it because I thought that robust historical gazetteers (having spatial footprints) that we could switch to using were right around the corner, but that hasn’t happened. Phasing it out would mean revisiting the periods that link to it, and replacing the links with links to administrative entities or physical/geographic regions.

Thoughts @atomrab?

**Comment by atomrab** [comment]
@rybesh -- Yes, this is a core issue that connects to the question of what a period definition is referring to: a concept applied to a place that has modern coordinates by modern scholars, or a phenomenon attached to a place that had coordinates in the past? We used modern national boundaries as our default at the beginning because period discourse was shaped by national boundaries and shared languages, so the "Roman" period differed from modern place to modern place, usually corresponding to differences from modern language to modern language. Plus we had geometry we could put on a map. The Pleiades question #41 reflects an attempt to get at this question of usage more directly, by mapping out all the sites where scholars had applied a specific meaning of a period term.

But then we have problems with larger synthetic thesauri/gazetteers like the Getty or LCSH that want to use one value for "Roman" or "Early Byzantine" across the area of a past state, without saying where they think that past state is. For LCSH, we tried to deal with this by providing all of the modern nation-states that overlapped with the territory of the ancient political entity (mainly only for the Roman and Byzantine empires), since those are the places that LCSH seemed to be applying the period term to. But this raises issues of both time and authorial intent -- if the term is "Roman Republican--Rome", does that mean only the city? Only the parts of modern Italy controlled by Rome before the 1st c BCE? Etc.

I've used both approaches, once we had a historical gazetteer -- it's a lot easier on a data-entry level to connect to an entry for "Roman Empire" than listing every modern nation within its borders (and you get to punt on the question of "borders when?"). But I'm not very satisfied with the value this adds, without geometry you can represent on a map. So I agree that (1) is the least desirable solution, until someone gives us a gazetteer with some solid historical geometry shapefiles (I downloaded a bunch of these at one point and have put them in a Box folder [here](https://utexas.box.com/s/1x7j687mqsm4ey4h4uxrcrgdkl6p6ki7); these are from Klokan Technologies (for [the Roman world](https://www.klokantech.com/references/romanempire/)) and from the archival version of the ThinkQuest website -- but they don't have URIs and aren't verifiable in terms of quality). Looking at you, World Historical Gazetteer!

Of 2 and 3, I tend to prefer 2, even if it's more work, just to help avoid lots of false positives in map-based search results (and the misapplication of terms to places where they don't really apply -- like using Seleucid for Egypt, since Egypt is in the Middle East, but the Seleucids were never in control of it. What about a decision tree -- at least for our internal work, and specified in the guides -- that indicates you should use 2 if you can, and only go to 3 if you can't make a call on which current nation-states are involved? For instance, for Lan Xang, if you take a look at [the map associated with the Wikidata entry](https://www.wikidata.org/wiki/Q853477#/media/File:Map-of-southeast-asia_1400_CE.png), you see that the kingdom is in Laos and Thailand, but not Cambodia -- so I'd prefer to associate it with the Wikidata entries for Laos and Thailand, rather than Indochina.

The problem here is that this is a manual process with human reasoning involved. I'm not sure how it could be automated for the DAI/Getty dataset. It might be a little easier to switch "the Roman Empire" or "the Byzantine Empire" or "the Holy Roman Empire" from historical to admin entities, since I think we've done all of these in both ways. Could we create our own aggregations of modern countries that would be shorthand for each of these historical entities, so contributors wouldn't have to enter dozens of modern states each time?

One related issue that will come up soon is historical-site-specific chronologies. I expect that we'll be adding a chronology specific to Dura Europos in Syria this summer -- but not to all of Syria. I don't think we have a lot of individual sites in our gazetteer. Should we add these programmatically? On an ad-hoc basis, by request? Not at all? Here we're in better shape, because there is a Wikidata entry with point geometry (can we work with points?), but the individual archaeological site can fall between "modern place" and "historical place" if a given site isn't recorded or geolocated in Wikidata.
